src/composer.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@tbillington `composer.json` should be in the root dir. Also refer to `.gitignore` changes above which should move back to root dir (unless you did it for a particular reason and I'm missing something?).